toonz/sources/common/trop/brush.cpp

```diff
@@ -1,4 +1,7 @@
 
+#ifdef LINUX
+#include <cstring>
```

Comment:
I tried a build also on a debootstrapped Debian/testing and had the same error (as sid/unstable), `-fno-implicit-templates` seems to be the big deal to get it linking.